### ISSUE TITLE
Logger only logs the `eventType` and not the properties

### ISSUE THREAD

**Issue by thsaravana** [amplitude/Amplitude-Android#150]
When I enable logging, so that I can see Amplitude logs in Logcat, I see only `eventType`. The `eventProperties`, `apiProperties`, `userProperties`, `groups` are not logged.

It would be great to see these in the log as well.

**Comment by autonomousapps** [comment]
Good point. In my project, we have do custom logging to see those properties. If Amplitude did that itself, we could remove our (then-)redundant code.

**Comment by thsaravana** [comment]
I saw the code and I would love to work on it and submit a pull request (i.e, if you guys haven't planned to work on this).

**Comment by haoliu-amp** [comment]
@thsaravana Definitely, we welcome any PRs. This year we will be putting more focus on SDKs. Sorry for the delay of replying.
